config/config-package.py

```diff
@@ -31,6 +31,11 @@ def copy_with_meta(template_name, destination, config_type, **kw):
     description='Use configuration for a package.')
 parser.add_argument(
     'path', type=pathlib.Path, help='path to the repository to be configured')
+parser.add_argument(
+    '--no-commit',
+    dest='no_commit',
```

Comment:
We could use the chance to switch away from the negated form when storing the value to a variable as it requires a double negation later on (`if not args.no_commit`). See the `--without-legacy-python` as an example. But the default here should be `true`.

This also applies to `--no-push`.

Reply:
How do you expect this to work? Do you want to replace `--no-commit` with something like `--commit=False`? That won't work with argparse action `store_false`, it will fail because you cannot pass an argument (`=False`). So you can only say `--commit`, which will do the `store_false` but it makes no sense.

Reply:
Got it after actually looking at `--without-legacy-python`, sorry. Take a look and let me know what you think.